### PR TITLE
uadk - the description document of the uadk log is added

### PR DESCRIPTION
**Pull request by yekai123123**

### CODE REVIEW

**Comment by gaozhangfei** [comment]
Do we need to update script/sanity.sh?
How about adding one doc describing the test cmd, and adding more later?
docs/log.rst need to be checked grammar.

File: docs/log.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+   local5.err        # display the error conditions
+   local5.info       # display the warning and error conditions
+   local5.debug      # display the debug,warning,error conditions
+   local5.*          # print levels are not differentiated.
+
+The following steps will help you set up the syslog file:
+
+step 1:
+   Add the following information to the last line of /etc/rsyslog.conf:
+   local5.err                                         /var/log/uadk.log
```
Comment (gaozhangfei) [inline docs/log.rst:24]: how about using diff here, maybe more clear.

File: docs/log.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+The rsyslog.conf file is the main configuration file for the rsyslogd which logs system
+messages on systems. The uadk library can set up the log level and log file by use the rsyslogd.
+the uadk log can be printed in message or syslog file default. You can set your own uadk.log
+file in rsyslog.conf.
+
+1. If selected the static compilation, the syslog will not be used, The information will be printed
+   on the serial port.
+2. If not edited the rsyslog.conf, the log files will be printed in /var/log/messages.
+3. If you don't want to use the rsyslogd, you can edit the Makefile.am. The information will be
```
[the remark this answers is not in the document]
Comment (yekai123123) [reply docs/log.rst:10]: Yes, the kernel use the rsyslogd default.

File: docs/log.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
+The rsyslog.conf file is the main configuration file for the rsyslogd which logs system
+messages on systems. The uadk library can set up the log level and log file by use the rsyslogd.
+the uadk log can be printed in message or syslog file default. You can set your own uadk.log
+file in rsyslog.conf.
+
+1. If selected the static compilation, the syslog will not be used, The information will be printed
```
[the remark this answers is not in the document]
Comment (yekai123123) [reply docs/log.rst:7]: Yes, like ./config.sh --static

File: docs/log.rst (hunk 1 above)
Comment (yekai123123) [reply docs/log.rst:24]: I think it's clear currently, Maybe it would be more clear as use the Markdown.

**Review by Liulongfang — APPROVED** [review]
Looks good to me

**Comment by gaozhangfei** [comment]
+1

**Pull request merged**